docs/website/content/integrations/lula.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Lula
+subtitle: The Cloud-Native Compliance Engine
+software:
+- lula
```

Comment:
While there are lots of counter examples to this on the site still, ideally we have softwares for integrations with _other_ software. E.g. all these projects integrate Terraform and OPA. https://www.openpolicyagent.org/softwares/terraform/

```suggestion
```

Sorry if the docs and other listings here aren't leading by example here... but in an effort for `/integrations/lula/` to get a good page rank, having `/softwares/lula/` is not helping keep it focused.